package.json

```
@@ -1157,6 +1157,13 @@
 				"category": "IBM i",
 				"icon": "$(filter)"
 			},
+			{
+				"command": "code-for-ibmi.copyFilter",
+				"enablement": "code-for-ibmi:connected == true",
+				"title": "Copy filter",
+				"category": "IBM i",
+				"icon": "$(filter)"
```

I think it would be better to use the icon for copy instead of filter - to distinguish between create and copy:

```suggestion
				"icon": "$(copy)"
```